scripts/log_parser.py

```diff
@@ -0,0 +1,376 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2022 Valory AG
+#
```

Comment:
Annoying that we cannot see the diff to the old file

Reply:
Even if we could, the file  is basically 100% different.